praktikumsplaner-backend/src/main/resources/db/migration/schema/V21__use_uuid_col_type.sql

```
@@ -0,0 +1,11 @@
+ALTER TABLE praktikumsstelle DROP CONSTRAINT fk_praktikumsstelle_mz;
+ALTER TABLE praktikumsstelle DROP CONSTRAINT fk_praktikumsstelle_nwk;
+
+ALTER TABLE nwk ALTER COLUMN id TYPE UUID USING id::uuid;
+ALTER TABLE praktikumsstelle ALTER COLUMN id TYPE UUID USING id::uuid;
+ALTER TABLE praktikumsstelle ALTER COLUMN meldezeitraumID TYPE UUID USING meldezeitraumID::uuid;
+ALTER TABLE praktikumsstelle ALTER COLUMN assignedNwk TYPE UUID USING assignedNwk::uuid;
+ALTER TABLE meldezeitraum ALTER COLUMN id TYPE UUID USING id::uuid;
+
+ALTER TABLE praktikumsstelle ADD CONSTRAINT fk_praktikumsstelle_mz FOREIGN KEY (meldezeitraumID) REFERENCES meldezeitraum(id);
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)